app/mailers/notification_mailer.rb

```diff
@@ -8,6 +8,7 @@ def build_message
 
     @subject = "[pdc-describe] New Notification"
     @message = @work_activity.message
```

Comment:
```suggestion
```
Might be unused now?

Reply:
This is still used in the text portion of the email and still needed